lib/memory

```diff
@@ -1 +0,0 @@
-[nil, nil, nil, nil, nil, nil, nil, nil, nil]
```

Comment:
You can avoid these files reappearing with [a .gitignore file](https://help.github.com/articles/ignoring-files/)